src/components/JaegerIntegration/JaegerResults/transform.ts

```diff
@@ -0,0 +1,288 @@
+import _isEqual from 'lodash/isEqual';
+import _round from 'lodash/round';
+import moment from 'moment';
+import { KeyValuePair, Span, SpanData, JaegerTrace, TraceData } from '../../../types/JaegerInfo';
+
```

Comment:
It seems this file comes mainly from Jaeger console, correct?
Could you put a comment & link to the original? So that if we have to dig into this code, we know where it refers to

Reply:
+1
A reference would help to maintain & understand some decisions.